src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -118,14 +133,19 @@ public function deprecate(string $name, string $package, string $version, string
         $this->deprecatedParameters[$name] = [$package, $version, $message, $name];
     }
 
+    public function nonEmpty(string $name, string $message = 'A non-empty value for the parameter "%s" is required.'): void
+    {
+        $this->nonEmptyParameters[$name] = \sprintf($message, $name);
```

Comment:
What about not doing the sprintf? I feel like it's here only for the default message and that's it's going to be useless or even annoying when a message is given.

Reply:
I removed the default message since this is something you'd want to customize anyway